### PR TITLE
First version of tool

### PR DESCRIPTION
**Pull request by raphaellondner-okta**
Demonstrates how to use the /api/v1/users Okta API to create a new Okta user (with the C# SDK https://www.nuget.org/packages/Okta.Core.Client/)

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+﻿## Okta Authentication Flow C# Command Line Sample
+
+This project demonstrates the use of Okta Authentication and Session APIs with [Okta .NET SDK](https://github.com/okta/oktasdk-csharp)  through the most common [transaction state models](http://developer.okta.com/docs/api/resources/authn.html#transaction-state).
+
+Specifically, you will learn when to expect an Okta state token(which you get when the user is expected to perform an additional operation, such as resetting their password or going a second-factor authentication) vs. a session token (when the user is fully authenticated and an OKta session is ready to be created) and what to do with them depending on the authentication state you're in.
```
Comment (jmelberg-okta) [inline Readme.md]: nit: token (space) (which ...

Comment (jmelberg-okta) [inline Readme.md]: OKta -> Okta

File: Readme.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+### Configuration
+Follow the steps below to set up the sample:  
+1. Edit the app.config file and uncomment the 4 lines in the `appSettings` section. 
+2. Create an Okta developer organization at [http://developer.okta.com](http://developer.okta.com).  
+3. Paste the url of your Okta developer organization (the one from the main user dashboard, not the one from the Admin dashboard with the "-admin" suffix in the url) into the `OktaTenantUrl` of your app.config file
+3. (Optional) Go to Admin -> Security -> API and press the "Create Token" button to create an API token. The API token is required if you want the code sample to call the Okta [Sessions API](http://developer.okta.com/docs/api/resources/sessions.html) in order to renew the current user's session.
```
Comment (jmelberg-okta) [inline Readme.md]: It would make more sense if we made this a requirement as opposed to an option. It will open up the entire feature set of your toolkit

**Comment by raphaellondner-okta** [comment]
@rchild-okta can you please review and provide feedback?

**Pull request merged**